Header header

std_msgs/Bool isOnTable
std_msgs/String label
